tests/backup/backup_test.go

```diff
@@ -6409,17 +6411,17 @@ var _ = Describe("{DeleteUsersRole}", func() {
 				dash.VerifyFatal(err, nil, fmt.Sprintf("Removing role [%s] from the user [%s]", role, userName))
 			}
 		})
-		// Waiting for keycloak to sync up before moving for validation
-		time.Sleep(time.Minute * 1)
 		Step("Validate if the roles are deleted from the users ", func() {
 			result := false
 			for user, role := range userRoleMapping {
-				roles, _ := backup.GetRolesForUser(user)
+				roles, err := backup.GetRolesForUser(user)
+				log.FailOnError(err, "Failed to get roles for user - %s", user)
 				for _, roleObj := range roles {
 					if roleObj.Name == string(role) {
 						result = true
 						break
 					}
+					log.Info(fmt.Sprintf("Role name from user [%s] added role [%s] user [%s]", roleObj.Name, string(role), user))
```

Comment:
This log message is not clear, needs to be re-worded

```
Role name from user [px-backup-app.user] added role [px-backup-infra.admin] user [testautouser1675943288]
```

Reply:
resolved